src/components/ReportGenerator/Report/Report.js

```diff
@@ -47,7 +48,7 @@ export const Report = ({ reportData, dataLoaded, paramsObject }) => {
   };
 
   const onDownloadHandler = () => {
-    downloadReport(paramsObject.current)
+    downloadReport(paramsObject?.current)
```

Comment:
If `downloadReport` should accept undefined, then `formatReportUrl` in `functions.js` should be adjusted to handle undefined for the `params` argument, otherwise a type error will be thrown.